code/mod_table/mod_table.cpp

```diff
@@ -448,6 +449,11 @@ void parse_mod_table(const char *filename)
 				stuff_boolean(&HUD_shadows);
 			}
 
+		if (optional_string("$Unify scanning behavior:")) {
+			stuff_boolean(&Use_new_scanning_behavior);
+		}
+
+		optional_string("#SEXP SETTINGS");
```

Comment:
Uh, this seems like it got here by accident

Reply:
Ok I just kinda redid things manually. I think everything's addressed now including this conflict resolution error